lib/inline_svg/action_view/helpers.rb

```diff
@@ -12,7 +12,7 @@ def inline_svg(filename, transform_params={})
             configured_asset_file.named filename
           end
         rescue InlineSvg::AssetFile::FileNotFound
-          return "<svg><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
+          return "<svg class='#{InlineSvg.configuration.svg_not_found_css_class}'><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
```

Comment:
I wonder if it would be better to only add the CSS class when it has been configured by a user? 🤔 

This would leave the default behavior unchanged (no CSS class), which would be less risky for existing users and should fix any unrelated test failures.

What do you think?